docs/guides/build_and_publish.md

```diff
@@ -1,19 +1,21 @@
-# Publishing a package
+# Building and publishing a package
 
-uv supports building Python packages into source and binary distributions via `uv build` and
-uploading them to a registry with `uv publish`.
+uv supports building Python packages into source and binary distributions via the `uv` build
+backend, the `uv build` build frontend and uploading them to a registry with `uv publish`.
 
-## Preparing your project for packaging
-
-Before attempting to publish your project, you'll want to make sure it's ready to be packaged for
-distribution.
+## Building your package
 
 If your project does not include a `[build-system]` definition in the `pyproject.toml`, uv will not
 build it by default. This means that your project may not be ready for distribution. Read more about
 the effect of declaring a build system in the
-[project concept](../concepts/projects.md#build-systems) documentation.
+[project concept](../concepts/projects.md#build-systems) documentation. To use uv as build backend,
+add the following to `pyproject.toml`:
```

Comment:
I think it's worth mentioning here that `uv` supports other build backends as well.